bench/wtperf/wtperf.c

```diff
@@ -503,11 +502,8 @@ worker(void *arg)
 		}
 	}
 	/* Setup the timer for throttling. */
-	if (thread->workload->throttle != 0 &&
-	    (ret = __wt_epoch(NULL, &interval)) != 0) {
-		lprintf(cfg, ret, 0, "Get time call failed");
-		goto err;
-	}
+	if (thread->workload->throttle != 0 && setup_throttle(thread) != 0)
+			goto err;
```

Comment:
Also, normal style is to propagate out all errors.  In other words, change this to:

```c
	if (thread->workload->throttle != 0 &&
	    (ret = setup_throttle(thread)) != 0)
		goto err;
```